doc/baremetal/hostlink.rst

```diff
@@ -21,11 +21,15 @@ a need to provide implementation of ``_exit()`` either through linking with some
 libgloss implementation or by implementing it right in the application.
 
 Because libgloss implementation is often specific to a particular chip (nee
-:abbr:`BSP (board support package)`), Synopsys provides only two libgloss
-implementation as of now - libnsim, that supports nSIM GNU IO Hostlink and
-libnosys, which is really an architecture-agnostic implementation, that simply
-provides empty stubs for a few of the most used system functions - this is
-enough to link an application, however it may not function as expected.
+:abbr:`BSP (board support package)`), Synopsys provides several libgloss
+implementations to cover as much usecases as possible.
+
+Two of them are for different Hostlink implementations, mechanism which allows
+target programm to invoke system call on host. One is libnsim, that supports
+nSIM GNU IO Hostlink and the other is libhl, that supports Metaware Hostlink.
```

Comment:
Please separate *libnsim* and *libhl* descriptions by numbers:

1. libnsim ....
2. libhl ...

Reply:
I'd also suggest to enclose library names in single quotes so they stand-out of the rest of the text like `libnsim`, `libhl`.